orbit/pkg/update/notifications.go

```diff
@@ -397,3 +398,86 @@ func (h *runScriptsConfigFetcher) GetConfig() (*fleet.OrbitConfig, error) {
 	}
 	return cfg, err
 }
+
+type BitlockDiskEncryptionPayload interface {
+	SetOrUpdateDiskEncryptionPayload(diskEncryptionStatus fleet.OrbitHostDiskEncryptionKeyPayload) error
```

Comment:
I'd remove the `Payload` part of the interface and method names, this is typically used for structs in our codebase to represent data, and in this case this is an interface for an HTTP client - I'd suggest `DiskEncryptionKeySetter` for the interface and `SetOrUpdateDiskEncryptionKey` as the method name (to match the name of the `Service` method for the corresponding endpoint from my PR: https://github.com/fleetdm/fleet/pull/14004/files#diff-359e1932f694ba39b5aefabce3730a8413e33764762fae5fb57d43bbc4f32f3bR794).